src/ec/packages/ec-component-message/message.story.js

```diff
@@ -15,6 +19,40 @@ import dataWarning from './demo/data--warning';
 import message from './ecl-message.html.twig';
 import notes from './README.md';
 
+const iconsList = {};
+iconsList.none = null;
+
+notifIcons.forEach(icon => {
+  iconsList[icon] = icon;
+});
+
+const PrepareMessage = data => {
+  const name = select(
```

Comment:
Mmhh, i think this should not be editable, we have all the stories for all the variants, therefore this should be a select with the only option being the current icon.name. Also because this at the moment doesn't make much sense, you can basically apply the wrong icon to a certain variant.